client/my-sites/upgrades/domain-management/edit/mapped-domain.jsx

```diff
@@ -110,10 +111,11 @@ const MappedDomain = React.createClass( {
 
 		return (
 			<VerticalNavItem path={ path }>
-				{ this.translate( 'DNS Records' ) }
+				{ this.props.translate( 'DNS Records' ) }
 			</VerticalNavItem>
 		);
 	}
 } );
 
-module.exports = MappedDomain;
+export { MappedDomain };
```

Comment:
I'm grateful that this fixes the test but.... _why_? 😅 Wasn't the whole idea of `localize` that it eased testing? Maybe it's the test that should be fixed? (though, I can't see how either :/)